scripts/deep.js

```diff
@@ -54,7 +54,14 @@ function getPath (obj, ks) {
 // Credit: https://github.com/jashkenas/underscore/blob/master/underscore.js
 function powerPick (object, keys) {
   var result = {}, obj = object, iteratee;
-  iteratee = function(key, obj) { return key in obj; };
```

Comment:
ok, I think I found a solution that handles all kind of nested data, but uses an extra module.

so I replaced powerPick with this:
```
function powerPick(object, keys) {
  var result = {};

  for (var i = 0, length = keys.length; i < length; i++) {
    var key = keys[i];
    result[key] = resolvePath(object, key);
  }

  return result;
}
```

where `resolvePath` is 
```
var resolvePath = require('object-resolve-path');
```

and `object-resolve-path` is a package that resolves all kind of values of nested objects, identified by a string key: https://github.com/capaj/object-resolve-path

does this work for you? how can I help to get this out?